raiden/ui/console.py

```diff
@@ -216,27 +224,6 @@ def register_token(self, token_address_hex):
         self._raiden.register_channel_manager(channel_manager.address)
         return channel_manager
 
-    def ping(self, peer_address_hex, timeout=4):
```

Comment:
Or, provide a simple interface for users to the health check, that allows quick ad hoc examination of the connection.